src/view/util/CaseDataInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+interface Props {
+  lastUpdated?: string;
+}
+
+const CaseDataInfo: React.FC<Props> = props => {
+  if (!props.lastUpdated) {
+    return null;
+  }
+
+  if (Number.isNaN(Date.parse(props.lastUpdated))) {
+    console.error(`Invalid date: ${props.lastUpdated}`);
+    return null;
+  }
+
+  const caseAgeEnd = new Date(props.lastUpdated);
+  const caseAgeStart = new Date(caseAgeEnd);
+  caseAgeStart.setFullYear(caseAgeEnd.getFullYear() - 1);
+
+  return (
+    <React.Fragment>
+      <p>
+        Showing cases from: {caseAgeStart.toLocaleString()} -{" "}
```

Comment:
If I'm understanding this correctly, it's telling the users that we are showing cases created in the last year, which is not quite correct. 🙂

Reply:
🙃 Yeah this is not right at all. Will fix